src/arr/locker/~tables.arr

```diff
@@ -0,0 +1,86 @@
+#lang pyret/library
+
```

Comment:
why is this file in this pr? i presume ~foo style files are some editor induced file?